iocage_lib/ioc_start.py

```diff
@@ -159,6 +159,7 @@ def __start_jail__(self):
         debug_mode = True if os.environ.get(
             'IOCAGE_DEBUG', 'FALSE') == 'TRUE' else False
         assign_localhost = self.conf['assign_localhost']
+        localhost_ip = self.conf['localhost_ip']
```

Comment:
Automatically generating that is problematic. There are a few issues with that. We'd have to change the jails hosts file each time it boots, which could impact services they have listening for/on these IP's. The safest approach is to generate it once, do our best to ensure uniqueness on the host at that time, and set the property for them to change later. We should avoid changing things inside the jail as often as we can, it would break POLA otherwise.